.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
+
+script:
+  - pycodestyle nyaa/ --show-source --max-line-length=100
```

Comment:
I think you should call `./lint.sh` and fix the script call in that file, since the file already exists.